src/utils/sql.ts

```diff
@@ -134,14 +134,14 @@ export function injectReplacements(
     if (isNamedReplacements && char === ':') {
       const previousChar = sqlString[i - 1];
       // we want to be conservative with what we consider to be a replacement to avoid risk of conflict with potential operators
-      // users need to add a space before the bind parameter (except after '(', ',', and '=')
-      if (previousChar !== undefined && !/[\s(,=]/.test(previousChar)) {
+      // users need to add a space before the bind parameter (except after '(', ',', and '=', '[' (for arrays))
+      if (previousChar !== undefined && !/[\s(,=[]/.test(previousChar)) {
```

Comment:
`and` shall be after `'='` and before `'['`

Reply:
Oh yeah, good catch